project.godot

```diff
@@ -35,7 +35,11 @@ SimulationParameters="*res://simulation_parameters/SimulationParameters.cs"
 Invoke="*res://src/engine/Invoke.cs"
 GUICommon="*res://src/gui_common/GUICommon.cs"
 SceneManager="*res://src/engine/SceneManager.cs"
-ScreenOverlays="*res://src/engine/ScreenOverlays.tscn"
+SaveStatusOverlay="*res://src/saving/SaveStatusOverlay.tscn"
+ColourblindScreenFilter="*res://src/engine/ColourblindScreenFilter.tscn"
+LoadingScreen="*res://src/engine/LoadingScreen.tscn"
+FPSCounter="*res://src/engine/FPSCounter.tscn"
+OverlayManager="*res://src/engine/OverlayManager.cs"
```

Comment:
I think you probably accidentally committed these changes

Reply:
Should be good now